presto-main/src/main/java/com/facebook/presto/sql/planner/sanity/VerifyNoFilteredAggregations.java

```diff
@@ -42,8 +42,8 @@ public Void visitAggregation(AggregationNode node, Void context)
         {
             super.visitAggregation(node, context);
 
-            for (FunctionCall call : node.getAggregations().values()) {
-                if (call.getFilter().isPresent()) {
+            for (Aggregation aggregation : node.getAssignments().values()) {
```

Comment:
see: `com.facebook.presto.sql.planner.sanity.VerifyNoFilteredAggregations` for instance. It checks that there are no aggregation nodes left with filter. You could do similar think for `FunctionCall` (e.g: check no `FunctionCall` expressions with `ORDER BY` clause are left).

However. I see that there is no such check for window function, so I think we could skip the check